web-ng/src/app/components/layer-dialog/layer-dialog.component.ts

```diff
@@ -184,9 +186,22 @@ export class LayerDialogComponent implements OnDestroy {
         : Map<string, Form>()
     );
 
+    if (this.projectId === ':new') {
+      this.projectService
+        .createProject(/* title= */ '')
+        .then((projectId: string) => {
```

Comment:
Can this be replaced with `projectId: => {`? (i.e. inferred type?)

Reply:
Done.